### DISCUSSION TITLE
Add Airplay support to KSVideoPlayerView

### DISCUSSION THREAD

**Discussion by mazz** [kingslay/KSPlayer#656] [General]
On iOS 17, SwiftUI, how do I add an airplay button along with airplay support to KSVideoPlayerView?

I would also like to add a fullscreen button ...

**Comment by kingslay** [comment]
KSAVPlayer 才能用airplay。我的swiftui demo如果判断播放器是KSAVPlayer 的话，那会自动加airplay。 swiftUI一般是自动转屏的，我还没找到方法来实现跟UIKit一样的强制转屏。

**Reply by mazz** [reply]
Thanks for the reply. I got it working. I didn't realize I needed to access Airplay from the lock screen
![IMG_7A6FECAF393D-1](https://github.com/kingslay/KSPlayer/assets/808934/6b9c87a7-df6e-471b-a223-d019d48eb818)
